scipy/stats/_stats_py.py

```diff
@@ -1606,23 +1608,29 @@ def skewtest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
     agree fairly closely, even for our small sample.
 
     """
+    xp = array_namespace(a)
+    a, axis = _chk_asarray(a, axis, xp=xp)
+
     b2 = skew(a, axis)
     n = a.shape[axis]
     if n < 8:
         raise ValueError(
-            "skewtest is not valid with less than 8 samples; %i samples"
-            " were given." % int(n))
+            f"skewtest is not valid with less than 8 samples; {n} samples were given.")
```

Comment:
Do we need this?

```suggestion
            f"skewtest is not valid with less than 8 samples; {int(n):i} samples were given.")
```

Reply:
Not sure I can quite follow the syntax of what this is trying to do?

Reply:
Just wondering why the original included `int(n)`. Is it fine to drop it?

Reply:
This is a quirk of history as n used to be a float, see https://github.com/scipy/scipy/commit/226155655fb703f007497a88e173f45e05905dc1
![image](https://github.com/scipy/scipy/assets/60778417/7411d421-d770-4e7a-b455-a6a81175aa03)